Volume/Views/PublicationDetail/PublicationDetailHeader.swift

```diff
@@ -10,17 +10,24 @@ import SwiftUI
 
 struct PublicationDetailHeader: View {
     @EnvironmentObject private var userData: UserData
+    @State private var hasOddNumberOfTaps = false
```

Comment:
I honestly think it makes more sense to have "hasOddNumberOfTaps."  Otherwise, what would constitute true for "didToggleFollowed"?  Maybe if pressed once or twice it makes sense but what if a user presses the button 9 times?  Would it make sense for a variable named "didToggleFollowed" to be true or false there? I'm honestly not sure, whereas it would make sense for "hasOddNumberOfTaps" to definitely be False.  This could just be me but I think it makes more sense the first way.